system/security/LdapSecurity/permissions.cpp

```diff
@@ -77,10 +77,10 @@ unsigned PermissionProcessor::ldap2sec(unsigned ldapperm)
     if(ldapperm & 0x28)
         permission |= SecAccess_Write;
```

Comment:
There is not an enum for any combination of permissions, its a bitfield and needs to be examined as such